extensions/amp-analytics/0.1/events.js

```diff
@@ -1487,7 +1487,9 @@ export class VisibilityTracker extends EventTracker {
           visibilityManager => {
             return visibilityManager.listenRoot(
               visibilitySpec,
-              this.getReadyPromise(waitForSpec, selector),
+              this.getReadyPromise(
+                this.getWaitForSpecForRootSelector_(waitForSpec, selector)
```

Comment:
What about inline the function? It's only called here once.

Reply:
Sg